client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -350,4 +363,30 @@ PreviewFrame.defaultProps = {
   cmController: {}
 };
 
-export default PreviewFrame;
+const mapStateToProps = state => ({
+  files: state.files,
```

Comment:
You're missing a bunch of props here. The ones that were passed via IDEView are:
```js
                    htmlFile={this.props.htmlFile}	
                    files={this.props.files}	
                    content={this.props.selectedFile.content}	
                    isPlaying={this.props.ide.isPlaying}	
                    isAccessibleOutputPlaying={	
                      this.props.ide.isAccessibleOutputPlaying	
                    }	
                    textOutput={this.props.preferences.textOutput}	
                    gridOutput={this.props.preferences.gridOutput}	
                    soundOutput={this.props.preferences.soundOutput}	
                    setTextOutput={this.props.setTextOutput}	
                    setGridOutput={this.props.setGridOutput}	
                    setSoundOutput={this.props.setSoundOutput}	
                    dispatchConsoleEvent={this.props.dispatchConsoleEvent}	
                    autorefresh={this.props.preferences.autorefresh}	
                    previewIsRefreshing={this.props.ide.previewIsRefreshing}	
                    endSketchRefresh={this.props.endSketchRefresh}	
                    stopSketch={this.props.stopSketch}	
                    setBlobUrl={this.props.setBlobUrl}	
                    expandConsole={this.props.expandConsole}	
                    clearConsole={this.props.clearConsole}	
                    cmController={this.cmController}	
                    language={this.props.preferences.language}
```